CoreFoundation/NumberDate.subproj/CFTimeZone.c

```diff
@@ -1319,9 +1328,8 @@ Boolean _CFTimeZoneInit(CFTimeZoneRef timeZone, CFStringRef name, CFDataRef data
     CFRelease(abbrevs);
 
     if (tzName) {
-        int32_t offset;
         __CFTimeZoneGetOffset(tzName, &offset);
-        // TODO(compnerd) handle DST
+        // TODO: handle DST
```

Comment:
I'm thinking here we might not need to care about handling DST. The CF functions which expose DST functionality accept an absolute time which they use to check against. Since we're not given a date here anyways, the correct thing to do might be to ignore it until given a date.